### ISSUE TITLE
Unable to reproduce COCO dataset CLIPScore and FID results for Stable Diffusion and FLUX

### ISSUE THREAD

**Issue by betacatZ** [maple-research-lab/TPDM#6]
I am trying to reproduce the evaluation results of Stable Diffusion and FLUX on the COCO dataset as reported in your paper.
However, my reproduced results on CLIPScore and FID do not match the reported ones. Could you provide the code or more details to help me reproduce the results?

## Settings
**dataset**: coco2017 validation set, for each image, I select `captions[0]` as the prompt.

**Evaluation metrics**:
   - **CLIPScore**: Computed using the method from [linzhiqiu/t2v_metrics](https://github.com/linzhiqiu/t2v_metrics) (openai:ViT-L-14-336)
   - **FID**: Computed using `pytorch-fid`

**Inference settings**

**stable-diffusion-3-medium-diffusers**
- num_inference_steps: 28
- guidance_scale: 7.0

**FLUX.1-dev**
- num_inference_steps: 28
- guidance_scale: 3.5

## Results

| Model                               | CLIPScore  | FID   |
|-------------------------------------|------------|-------|
| stable-diffusion-3-medium-diffusers | 0.2655     | 30.66 |
| FLUX.1-dev                          | 0.2608   | 32.08 |
